### ISSUE TITLE
Lyra doesn't find type definitions when not run from 'root'

### ISSUE THREAD

**Issue by ahpook** [lyraproj/lyra#262]
**Describe the bug**
When creating and applying workflows from outside the build directory, the workflows are found but Lyra errors with `Reference to unresolved type Type::Name`

**To Reproduce**
1. Create a directory containing a `data.yaml` and `workflow/workflow.yaml` and run a docker container with the directory bind-mounted into the `local/` subdirectory in the container
```
docker run -it \
   --mount type=bind,src=$HOME/Sandbox/lyra-local,dst=/src/lyra/local \
   lyraproj/lyra:latest /bin/ash
```

Inside the container, cd into that `local` directory and attempt to apply the workflow:

```
/src/lyra # cd local
/src/lyra/local # lyra apply azure_wordpress
rpc error: code = Unknown desc = Reference to unresolved type 'AzureRM::Resource_group' (file: /go/pkg/mod/github.com/lyraproj/puppet-workflow@v0.0.0-20190507102935-c5a6d8ad7529/yaml/activity.go, line: 566)
```

The type is found correctly if you copy the workflow into the top-level `workflows` directory, but this is undesirable because it won't persist after container exits, and when running from a git checkout or package distribution, it means you're mixing local changes with the upstream codebase.

**Expected behavior**
Users should be able to have a "working" area containing hiera data, `workflows/` and ts/go code that is independent of the distributed binary / source tree, so they can work locally without "polluting" either their repo checkout or the distribution tree.

**Comment by ahpook** [comment]
I have a sample repo to clone down: https://github.com/ahpook/lyra-local

**Comment by thallgren** [comment]
Fixed by lyraproj/puppet-workflow#23 and #267.
